draft-regext-rdap-x-media-type.md

```diff
@@ -139,8 +134,19 @@ passed from the client to the server, and the presence of these protocol element
 used by servers to determine a client's capability to handle the RDAP extension. This specification
 does not require the usage of those extension identifiers in the "extensions" parameter,
 though clients SHOULD list the extension identifier in the "extensions" parameter when using
-other protocol elements of those extensions. Servers SHOULD NOT require the usage of extension
-identifiers in the "extensions" parameter when other extension protocol elements are used.
+other protocol elements of those extensions for better compatibility with servers
+recognizing the "extensions" parameter. Servers SHOULD NOT require the usage of extension
+identifiers in the "extensions" parameter when other extension protocol elements are used for
+backwards-compatibility purposes.
+
+## RDAP-X Extension
+
+This document defines an RDAP "profile" extension using the identifier "rdapx" (hyphen
```

Comment:
Knowing "rdapx" is pithy, but per the "yuck" comment from the media-types folks, would it be better to expand the extension id to "rdapExtensions", or "rdapExtensions1" (a bit yucky) in case another id in future for this profile extension?